oidc-controller/api/core/acapy/config.py

```diff
@@ -65,6 +68,62 @@ def get_headers(self) -> dict[str, str]:
         return {"Authorization": "Bearer " + self.get_wallet_token()}
 
 
+class TractionTenantAcapy:
```

Comment:
In Traction the multitenant tokens expire with a default of 1 day configurable. VCAuth processes/containers would be running past that (or the expiry could be very short if desired and still be shorter than the life of a serverless process runtime) would token failures trigger a re-fetch?